include/h2o.h

```diff
@@ -1085,6 +1085,11 @@ struct st_h2o_req_t {
      */
     H2O_VECTOR(h2o_req_error_log_t) error_logs;
 
+    /**
+     * proxy log data
+     */
+    h2o_http1client_timings_t *proxy_timings;
```

Comment:
Thank you for the changes.

How about adding `h2o_http1client_timings_t` (not a pointer) as a member of `h2o_req_t::timestamps`?

By doing so, I believe that we can drastically reduce the changes. For example, there would be no need for new functions like `h2o_proxy_time_compute_...`. All we need to do in core is to support new variables in logconf.c.